### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by itheiss**
Added an example how to specify an offset by configuration.

### CODE REVIEW

File: addons/binding/org.openhab.binding.astro/README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ No binding configuration required.
 
 ## Thing Configuration
 
-The things requires the geolocation (latitude, longitude) for which the calculation is done. Optionally, a refresh interval (in seconds) can be defined to also calculate positional data like azimuth and elevation.
+The things require the geolocation (latitude, longitude) for which the calculation is done. Optionally, a refresh interval (in seconds) can be defined to also calculate positional data like azimuth and elevation.
```
Comment (ThomDietrich) [inline addons/binding/org.openhab.binding.astro/README.md]: Alternatively "A thing requires"?

**Comment by ThomDietrich** [comment]
Hey @itheiss, thanks for adding this information! Looking at the result, I believe the document could use some reorganization. Move the events list and the offset description up and add the example to the existing Things example.

File: addons/binding/org.openhab.binding.astro/README.md (hunk 1 above)
Comment (itheiss) [inline addons/binding/org.openhab.binding.astro/README.md]: Sounds better...

**Comment by itheiss** [comment]
Hi @ThomDietrich. Updated as suggested. What do you think?

File: addons/binding/org.openhab.binding.astro/README.md [hunk 2]
```diff
@@ -49,37 +49,7 @@ The things require the geolocation (latitude, longitude) for which the calculati
...
-
-```
-DateTime Sunrise_Time  "Sunrise [%1$tH:%1$tM]"  { channel="astro:sun:home:rise#start" }
-DateTime Sunset_Time   "Sunset [%1$tH:%1$tM]"   { channel="astro:sun:home:set#start" }
-Number   Azimuth       "Azimuth"                { channel="astro:sun:home:position#azimuth" }
-Number   Elevation     "Elevation"              { channel="astro:sun:home:position#elevation" }
-String   MoonPhase     "MoonPhase"              { channel="astro:moon:home:phase#name" }
-```
-
 Events:
```
Comment (ThomDietrich) [inline addons/binding/org.openhab.binding.astro/README.md]: The correct term would actually be `### Trigger Channels`

File: addons/binding/org.openhab.binding.astro/README.md [hunk 3]
```diff
@@ -101,14 +71,44 @@ Available events:
 
 Offsets:
```
Comment (ThomDietrich) [inline addons/binding/org.openhab.binding.astro/README.md]: This one word is a bit lost here. How about `**Offsets:** For each...`

File: addons/binding/org.openhab.binding.astro/README.md [hunk 4]
```diff
@@ -101,14 +71,44 @@ Available events:
...
 ```
 
+Items:
+
+```
+DateTime Sunrise_Time  "Sunrise [%1$tH:%1$tM]"  { channel="astro:sun:home:rise#start" }
+DateTime Sunset_Time   "Sunset [%1$tH:%1$tM]"   { channel="astro:sun:home:set#start" }
+Number   Azimuth       "Azimuth"                { channel="astro:sun:home:position#azimuth" }
+Number   Elevation     "Elevation"              { channel="astro:sun:home:position#elevation" }
+String   MoonPhase     "MoonPhase"              { channel="astro:moon:home:phase#name" }
```
Comment (ThomDietrich) [inline addons/binding/org.openhab.binding.astro/README.md]: While you are at it, could your "correct" these items?
`"Azimuth [%.1f °]"`
`"Elevation [%.1f °]"`
`"Moon Phase [%s]"`

**Comment by itheiss** [comment]
Done. Hope I got you right with "break down paragraphs to one sentence per line".

File: addons/binding/org.openhab.binding.astro/README.md [hunk 5]
```diff
@@ -49,23 +50,57 @@ The things requires the geolocation (latitude, longitude) for which the calculat
...
 Things:
 
 ```
-astro:sun:home  [ geolocation="xx.xxxxxx,xx.xxxxxx", interval=60]
-astro:moon:home [ geolocation="xx.xxxxxx,xx.xxxxxx", interval=60]
+astro:sun:home  [ geolocation="xx.xxxxxx,xx.xxxxxx", interval=60 ]
+astro:moon:home [ geolocation="xx.xxxxxx,xx.xxxxxx", interval=60 ]
+```
+or optionally with an offset
+```
```
Comment (ThomDietrich) [inline addons/binding/org.openhab.binding.astro/README.md]: (The markdown engine at docs.openhab.org is a bit more strict) You need to always leave one empty line before and after a code block. That's why an empty line is needed before and after this sentence.

**Comment by kaikreuzer** [comment]
Thanks!

**Pull request merged**